collection_checklist.md

```diff
@@ -15,6 +15,7 @@ Every comment should say whether the reviewer expects it to be addressed, or whe
 - [ ] follows licensing rules
 - [ ] follows the [Ansible documentation standards](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html) and the [style guide](https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#style-guide)
 - [ ] follows [development conventions](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html)
+- [ ] supports Python 2.6 or greater and Python 3.5 or greater
```

Comment:
I think we should be more specific here.

python2.6 / python2.7 / python3.5 are all EOL. Expecting a new collection to support them, doesn't make too much sense.  I think it is fair to say these are optional, if you want to support them.

https://www.python.org/downloads/ could be a good format

Reply:
See https://github.com/ansible/ansible/issues/72668#issuecomment-733068389 which shows that some OS still support Python 2.7 till 2024

Reply:
I think 2.6+ and 3.5+ is the right set.